### PR TITLE
[RISCV] Added definition of Ventana veyron-v1 processor.

### PR DESCRIPTION
**Pull request by mgudim**

### CODE REVIEW

File: clang/test/Misc/target-invalid-cpu-note.c [hunk 1]
```diff
@@ -93,4 +93,4 @@
 
 // RUN: not %clang_cc1 -triple riscv64 -tune-cpu not-a-cpu -fsyntax-only %s 2>&1 | FileCheck %s --check-prefix TUNE-RISCV64
 // TUNE-RISCV64: error: unknown target CPU 'not-a-cpu'
-// TUNE-RISCV64-NEXT: note: valid target CPU values are: generic-rv64, rocket-rv64, sifive-s21, sifive-s51, sifive-s54, sifive-s76, sifive-u54, sifive-u74, sifive-x280, generic, rocket, sifive-7-series{{$}}
+// TUNE-RISCV64-NEXT: note: valid target CPU values are: generic-rv64, rocket-rv64, sifive-s21, sifive-s51, sifive-s54, sifive-s76, sifive-u54, sifive-u74, sifive-x280, generic, rocket, sifive-7-series{{$}}, veyron-v1
```
Comment (michaelmaitland) [inline clang/test/Misc/target-invalid-cpu-note.c:96]: I think this should go before the end of line marker?

Comment (mgudim) [reply clang/test/Misc/target-invalid-cpu-note.c:96]: Oh, right! Thank you!

**Comment by michaelmaitland** [comment]
> [Merge branch 'llvm:main' into mgudim_veyron_def](https://github.com/llvm/llvm-project/pull/65535/commits/454b41eea50a3583affffb5c29bffbd46bcd633b)

The [LLVM GitHub User Guide](https://llvm.org/docs//GitHub.html) recommends to rebase on main instead of merge main.

**Comment by mgudim** [comment]
> > [Merge branch 'llvm:main' into mgudim_veyron_def](https://github.com/llvm/llvm-project/pull/65535/commits/454b41eea50a3583affffb5c29bffbd46bcd633b)
> 
> The [LLVM GitHub User Guide](https://llvm.org/docs//GitHub.html) recommends to rebase on main instead of merge main.

Yes, my bad. Still learning the new workflow. I go this merge commit after pressing the "sync" button on my fork. Is there a way to fix this now?

**Comment by michaelmaitland** [comment]
> Is there a way to fix this now?

I think you can drop the merge commit using `git rebase -i`. You may have to pass `--rebase-merges` to have the ability to drop the merge commit. Then you can pull upstream and `git rebase upstream/main`.

**Comment by mgudim** [comment]
> I think you can drop the merge commit using `git rebase -i`. You may have to pass `--rebase-merges` to have the ability to drop the merge commit. Then you can pull upstream and `git rebase upstream/main`.

Thanks Michael, I tried but messed it up (I see more changes than I intended). Let me try again. If I mess it up again, is it OK if I just close this PR and start over?

**Comment by michaelmaitland** [comment]
> > I think you can drop the merge commit using `git rebase -i`. You may have to pass `--rebase-merges` to have the ability to drop the merge commit. Then you can pull upstream and `git rebase upstream/main`.
> 
> Thanks Michael, I tried but messed it up (I see more changes than I intended). Let me try again. If I mess it up again, is it OK if I just close this PR and start over?

I think you could drop [5fe9e82](https://github.com/llvm/llvm-project/pull/65535/commits/5fe9e82d849b85022a0a95016c65066dbae27e4d) using `git rebase -i`, recommit it, and then follow the LLVM GitHub User Guide on how to rebase on main. 

But if you want to close this PR and make another PR, thats fine by me.
